### PR TITLE
Gci 1367 validate scud

### PR DESCRIPTION
**Pull request by nabeelamirch**
validate scud item

resolves: GCI-1367

### CODE REVIEW

File: src/test/java/uk/gov/companieshouse/scanupondemand/orders/api/controller/ScanUponDemandItemControllerIntegrationTest.java [hunk 1]
```diff
@@ -270,6 +264,35 @@ void getScanUponDemandItemReturnsNotFound() throws Exception {
                 .andDo(MockMvcResultHandlers.print());
     }
 
+    @Test
+    @DisplayName("Fails to create scan upon demand item that fails validation")
+    void createScanUponDemandItemFailsToScanUponDemandItem() throws Exception {
```
Comment (VaughanJackson123) [inline src/test/java/uk/gov/companieshouse/scanupondemand/orders/api/controller/ScanUponDemandItemControllerIntegrationTest.java:269]: Test method name `createScanUponDemandItemFailsToScanUponDemandItem` is missing second `...Create...`?

**Comment by VaughanJackson123** [comment]
As it stands, GCI-1367 includes user authentication too. Is this going to be dealt with in a separate PR?

**Comment by nabeelamirch** [comment]
> As it stands, GCI-1367 includes user authentication too. Is this going to be dealt with in a separate PR?

I wasn't sure if it was necessary. The story says  the user is not authorised to file/logged in then a HTTP 401 response is returned. This happens already with ERIC.

**Comment by VaughanJackson123** [comment]
> I wasn't sure if it was necessary. The story says the user is not authorised to file/logged in then a HTTP 401 response is returned. This happens already with ERIC.

We ended up implementing authentication/authorisation for both of the other Item APIs. I would be surprised if this one turned out not to require something similar. Can you look into that? Perhaps the story needs aligning with the auth stories for those APIs, or we went overboard with the auth for those APIs?

**Comment by nabeelamirch** [comment]
> > I wasn't sure if it was necessary. The story says the user is not authorised to file/logged in then a HTTP 401 response is returned. This happens already with ERIC.
> 
> We ended up implementing authentication/authorisation for both of the other Item APIs. I would be surprised if this one turned out not to require something similar. Can you look into that? Perhaps the story needs aligning with the auth stories for those APIs, or we went overboard with the auth for those APIs?

There's no harm in adding it in, so I'll add it in

**Review by rjadhav-ch — APPROVED** [review]

**Comment by VaughanJackson123** [comment]
> There's no harm in adding it in, so I'll add it in

Hmm. I didn't mean to seem like I was insisting that we implement things exactly the same way here as was done in the other APIs. But it would be good to know, if we don't need all the extra AA checking here, why we seemed to need it for the other APIs. If you have verified these are just not required, then that's fine. We can create tickets to remove the unnecessary code from the other two APIs at a later date for instance?

**Review by hwilliams1 — APPROVED** [review]

**Review by VaughanJackson123 — APPROVED** [review]

**Pull request merged**